web/src/pages/home/homePage.js

```diff
@@ -595,7 +563,7 @@ const Homepage = () => {
         <div className="asia-countries-container">
           {asiaCountries.map((country, index) => (
             <div className="country-card asia-country-card" key={index}>
-              <a href="https://www.domits.com/home/" target="_blank" rel="noopener noreferrer">
+              <a href="https://www.domits.com/home/" rel="noopener noreferrer">
```

Comment:
Dit moet global zijn, niet specifiek naar domits.com
```suggestion
              <a href="https://www.domits.com/home/" rel="noopener noreferrer">
```